packages/cli/src/commands/doctor/healthchecks/__tests__/androidNDK.test.ts

```diff
@@ -0,0 +1,66 @@
+import androidNDK from '../androidNDK';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {NoopLoader} from '../../../../tools/loader';
+
+import * as common from '../common';
+
+const logSpy = jest.spyOn(common, 'logManualInstallation');
+
+describe('androidHomeEnvVariables', () => {
+  let initialEnvironmentInfo: EnvironmentInfo;
+  let environmentInfo: EnvironmentInfo;
+
+  beforeAll(async () => {
+    initialEnvironmentInfo = await getEnvironmentInfo();
+  });
+
+  beforeEach(() => {
+    environmentInfo = initialEnvironmentInfo;
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('returns a message if the Android SDK is not installed', async () => {
+    environmentInfo.SDKs['Android SDK'] = 'Not Found';
+    const diagnostics = await androidNDK.getDiagnostics(environmentInfo);
+    expect(typeof diagnostics.needsToBeFixed).toBe('string');
```

Comment:
This has actually been changed on #743, you can check the change for `androidHomeEnvVariable` [here](https://github.com/react-native-community/cli/pull/743/files#diff-9c043f20be368d1006196f0b0cb243c2R24).

Reply:
Just to clarify, this field was being used to hold a boolean and a string, the string was used as a `description` sorta of thing, on #743 I actually created the `description` field.

Reply:
Okay nice one. I like it like that!